core/src/qos_service.rs

```diff
@@ -0,0 +1,239 @@
+//! Producer quality of service
+//!
+//!
+use solana_measure::measure::Measure;
+use solana_runtime::{
+    bank::Bank,
+    cost_model::{CostModel, TransactionCost},
+    cost_tracker::CostTrackerError,
+};
+use solana_sdk::transaction::{self, SanitizedTransaction, TransactionError};
+use std::sync::{Arc, RwLock};
```

Comment:
Super nit-picky, but we're slowly transitioning the repo modules to use a single `use` statement. Can you reformat this?

Reply:
done